templates/kualitas_air/parameter.html

```diff
@@ -68,83 +73,133 @@ <h2><a href="/kualitasair/chart/$id_pos_ka"><span class="glyphicon glyphicon-che
   <div class="table-responsive">
     <table class="table table-sm table-hover table-bordered boldtable">
       <td style="background-color: yellow;">Parameter Uji</td>
+      <td style="background-color: yellow;">Satuan</td>
     $for d in waktu:
-        <td style="background-color: yellow;width: 80px;">$d</td>
+        <td style="background-color: yellow;width: 50px;">$d</td>
 
       <tr>
-        <td style="background-color: yellow;">Suhu (C)</td>
+        <td style="background-color: yellow;">Suhu</td>
+        <td style="background-color: yellow;">&#8451;</td>
         $for d in suhu:
-            <td>$d</td>
+            $if d == None:
```

Comment:
Bisa kau coba dengan statement seperti ini:

`<td>${ d != None and d or '-'}</td>`

Reply:
oke, saya test dulu pak